python/lsst/summit/utils/astrometry/fastStarTrackerAnalysis.py

```diff
@@ -451,36 +701,67 @@ def plotSourceMovement(results, sourceIndex=0, allowInconsistent=False):
         image sequence, and the second is a scatter plot of the x and y, with
         the color showing the position in the sequence.
     """
+    opts = {
+        "marker": "o",
+        "markersize": 6,
+        "linestyle": "-",
+    }
+
     consistent = checkResultConsistency(results.values(), silent=True)
     if not consistent and not allowInconsistent:
         checkResultConsistency(results.values(), silent=False)  # print the problem if we're raising
         raise ValueError("The sources were found to be inconsistent and allowInconsistent=False")
 
     sourceDict = {k: v[sourceIndex] for k, v in results.items()}
-    seqNums = list(sourceDict.keys())
+    frameNums = [s.frameNum for s in sourceDict.values()]
     sources = list(sourceDict.values())
 
+    allDayObs = set([s.dayObs for s in sources])
```

Comment:
You can drop the list `[]` since it'll work fine with a generator, FYI.

Reply:
Ah thanks, I always forget that for some reason!